### PR TITLE
converted to block subscription

### PR DESCRIPTION
**Pull request by ironoa**

### CODE REVIEW

File: src/subscriber.ts [hunk 1]
```diff
@@ -57,67 +51,85 @@ export class Subscriber {
...
+      const { signer, method: { args, method, section } } = extrinsic;
+      if(method != 'transfer' || section != 'balances') {
+        return
+      }
+      this.logger.debug(`received new transfer balances`)
+
+      const sender = signer
+      const receiver = args[0].toString()
+      const unit = args[1].toString()
+      this.logger.info(`sender: ${sender}\nreceiver: ${receiver}\nunit: ${unit}\nblockHash: ${blockHash}`)
```
Comment (fgimenez) [inline src/subscriber.ts:92]: We would get a log entry for every transfer in the block, no matter who the sender and receiver are, is this what we want?

File: src/subscriber.ts [hunk 2]
```diff
@@ -57,67 +51,85 @@ export class Subscriber {
...
     }
-  }
 
-  private _setTransactionType = (data: TransactionData, currentFreeBalance: Balance, previousFreeBalance: Balance, account: Subscribable): void => {
-    
-    // check if the action was performed by the account or externally
-    const change = currentFreeBalance.sub(previousFreeBalance);
-    if (!change.gt(ZeroBalance)) {
-        this.logger.info(`Action performed from account ${account.name}`);
+    private _transferBalancesExtrinsicHandler = async (extrinsic: Extrinsic, blockHash: string): Promise<void> =>{
```
Comment (fgimenez) [inline src/subscriber.ts:81]: It would be very nice to write a small unit test for this, maybe injecting a notifier stub, is that possible?

**Review by fgimenez — APPROVED** [review]
Looks good, only a couple of questions.

File: src/subscriber.ts (hunk 2 above)
Comment (ironoa) [reply src/subscriber.ts]: Done, thanks for the hint

**Review by pampatzoglou — APPROVED** [review]

**Pull request merged**